src/cppcache/src/CacheXmlParser.cpp

```diff
@@ -1165,7 +1166,7 @@ void CacheXmlParser::startPersistenceManager(const xmlChar** atts) {
         throw CacheXmlException(s.c_str());
       }
 
-      ACE_OS::strncpy(libraryFunctionName, (char*)atts[i], len);
+      std::strncpy(libraryFunctionName, (char*)atts[i], len);
```

Comment:
Alright.  Thanks for the clang-tidy tip.  I'll add these changes and update the pull request.